content/operator/cd/index.de.md

```diff
@@ -5,6 +5,11 @@ description: "Informationen zu den FIP-Bedingungen bei der České dráhy (ČD).
 country:
   - "czechia"
 operator: "cd"
+booking:
+  - "cd-website"
+  - "db-website-fip-db"
+  - "db-website-fip-international"
+  - "cd-ticket-office"
```

Comment:
I'm not sure if tracking the booking platforms in another place is a good idea. I'm sure that the list will get out-of-sync at some point. Instead, we could automatically derive the ticket offices by iterating over the operators and check if the raw content contains `booking id="%s"`. It's less performant during build time, but given the limited number of operators it shouldn't be an issue. What do you think about the idea?